src/output.h

```diff
@@ -0,0 +1,90 @@
+#pragma once
+
+#include <iostream>
+#include <iomanip>
+#include <vector>
+#include <map>
+
+#include "imap.h"
+
+namespace bpftrace {
+
+class Output
+{
+public:
+  explicit Output(std::ostream& out = std::cout, std::ostream& err = std::cerr) : out_(out),err_(err) { }
+  Output(const Output &) = delete;
+  Output& operator=(const Output &) = delete;
```

Comment:
Should probably add a virtual destructor for an abstract base class
```suggestion
  Output& operator=(const Output &) = delete;
  virtual ~Output() = default;
```